### ISSUE TITLE
Starting a Dev chain does not fund account Alice 

### ISSUE THREAD

**Issue by JesseAbram** [paritytech/polkadot#238]
Hey so I am having a little issue running a local Polkadot dev chain. The chain is running and I am connecting but the balance of Alice is 0.

node
```
Running target/debug/polkadot --dev --ws-port 9999 --rpc-port 9998 --ws-external --alice
Parity Polkadot
version 0.4.0-75e827c0-x86_64-linux-gnu

```

JS 
```
const keyring = new Keyring({ type: "sr25519" });

const alice = keyring.addFromUri('//Alice');
const previous = await api.query.balances.freeBalance(alice.publicKey());

```

**Comment by jacogr** [comment]
As a short-term work-around to have funds to play around with, you can add `//Bob//stash` (or Eve, ...) - since `//Bob` is not actually nominating/validating  on `--dev` the stash is not locked and available.

**Comment by JesseAbram** [comment]
Ya, I tried I have access to the account and I am able to return a tx hash but the balance to the stash account is still showing 0 and the balance of my the account I am sending to isn't receiving any of the funds

**Comment by chevdor** [comment]
From what I remember, `Bob` also has no balance. Neither do `Charlie`, `Eve`, etc...

**Comment by chevdor** [comment]
Note to self: The fix for this issue is *not included* in 0.4.4 but it works indeed in master.
